skills/session/scripts/rewind-session.py

```
@@ -151,6 +151,25 @@ def rewind_antigravity_db(db_path, cutoff_step, cid=None, summary_db_path=None,
+                    if not line.strip():
+                        continue
+                    data = json.loads(line)
+                    s_idx = data.get("step_index", 0)
+                    if s_idx <= effective_cutoff:
+                        st = data.get("type", "")
+                        if st == "USER_INPUT":
+                            user_turns += 1
+                        elif st == "PLANNER_RESPONSE":
+                            model_turns += 1
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)